castor/docker_events/models.py

```diff
@@ -0,0 +1,21 @@
+from __future__ import unicode_literals
+import json
+
+from django.contrib.postgres.fields import JSONField
+from django.db import models
+
+from docker_servers.models import DockerServer
+
+
+class DockerEvent(models.Model):
+    docker_server = models.ForeignKey(to=DockerServer)
+    capture_time = models.DateTimeField(auto_now=True)
+    data = JSONField(default={}, unique=True)
+
+    def __unicode__(self):
```

Comment:
The `__unicode__` method is not required in Python 3. You can use the `@python_2_unicode_compatible` decorator if you'd like to be Python 2 compatible.

[Reference](https://docs.djangoproject.com/en/1.10/topics/python3/#str-and-unicode-methods)